README.md

```diff
@@ -29,10 +30,10 @@
     - [Troubleshooting](#troubleshooting)
 - [Rebranding Notice](#rebranding-notice--loudspeaker-)
 - [Contributing](https://www.eclipse.org/jkube/contributing/)
+- [How to use Eclipse JKube snapshot artifacts?](./USING-JKUBE-SNAPSHOTS.md)
```

Comment:
```suggestion
- [How to use Eclipse JKube snapshot artifacts?](./USING-SNAPSHOT-ARTIFACTS.md)
```

Reply:
Sorry, I did a mistake while adding steps. There was a typo in entry (file name was wrong)

Reply:
No worries will update it and push it again.

Reply:
Updated from my end. Please check it and let me know if there is anything to be done.